### PR TITLE
Resolve spelling for nutraceuticals

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This just addresses all the new words that need to be added for the Nutraceuticals paper -- it's easier for me to do from the command line! It also fixes anything in the text itself that was caught by the spellchecker.
Finally, I alphabetized the dictionary because I noticed I had mixed up the order in a few places!

### Related issues

#748 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: build/assets/custom-dictionary.txt [hunk 1]
```diff
@@ -1,49 +1,672 @@
-personal_ws-1.1 en 1315 utf-8
```
Comment (agitter) [inline build/assets/custom-dictionary.txt:1]: This header line needs to appear at the top.  The 1315 is an approximate word count for the rest of the dictionary.

File: build/assets/custom-dictionary.txt [hunk 2]
```diff
@@ -1,49 +1,672 @@
...
+ABCpred
```
Comment (agitter) [inline build/assets/custom-dictionary.txt:1]: Do you want to sort in a case insensitive manner? I'm pretty sure @cgreene has a one-liner for dictionary updates that he shared in a different Manubot repository.

**Review by agitter — CHANGES_REQUESTED** [review]

File: build/assets/custom-dictionary.txt (hunk 2 above)
Comment (cgreene) [reply build/assets/custom-dictionary.txt:1]: https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/.github/PULL_REQUEST_TEMPLATE.md#spellcheck-step

File: build/assets/custom-dictionary.txt [hunk 3]
```diff
@@ -628,315 +985,171 @@ intubation
...
-PCR
-PDB
 peplomers
-PepSite
 perivasculitis
 permalink
 permeabilize
-Perrin
-Pfaender
+personal_ws-1.1 en 1315 utf-8
```
Comment (cgreene) [inline build/assets/custom-dictionary.txt:1152]: I think you will need to move this line to the top before runnign it

**Comment by rando2** [comment]
Thank you so much @agitter and @cgreene! Hopefully this fixes it 🤞

**Review by RLordan — APPROVED** [review]
Apologies, for the spelling mistakes! I made enough of them! 😂

**Review by agitter — APPROVED** [review]

**Pull request merged**